### PR TITLE
Brayns initialization modified

### PR DESCRIPTION
**Pull request by hernando**
Plugin loading now happens in the constructor, as it's needed to register
loaders from plugins before loading occurs (this will be the case of the
upcoming CircuitViewer plugin).

In braynsService, the event loop setup now must be done before creating
the Brayns instance.

### CODE REVIEW

File: apps/BraynsService/main.cpp [hunk 1]
```diff
@@ -156,7 +156,7 @@ class BraynsService
         _renderLoop->stop();
     }
 
-    brayns::Brayns _brayns;
+    std::shared_ptr<brayns::Brayns> _brayns;
```
Comment (rdumusc) [inline apps/BraynsService/main.cpp]: why not unique_ptr?

File: apps/BraynsService/main.cpp [hunk 2]
```diff
@@ -47,7 +46,14 @@ class BraynsService
         _setupMainThread();
         _setupRenderThread();
 
-        _brayns.loadPlugins();
+        _brayns.reset(new brayns::Brayns(argc, argv));
```
Comment (karjonas) [inline apps/BraynsService/main.cpp]:
```suggestion
        _brayns = std::make_unique<brayns::Brayns>(argc, argv);
```

**Comment by karjonas** [comment]
The copyright in DynamicLib.cpp should still be Intel, or did you change the code a lot?

**Comment by hernando** [comment]
Yes, I did

**Pull request merged**